biblioteka_filmów_zadanie7.4.py

```diff
@@ -0,0 +1,106 @@
+import faker
+import random
+from datetime import date
+
+today = date.today()
+d1 = today.strftime("%d.%m.%Y")
+
+fake = faker.Faker()
+
+
+class Movie:
```

Comment:
to zaimplementuj za pomocą dataclassy z modułu dataclasses: https://docs.python.org/3/library/dataclasses.html